.github/workflows/hugo_ci.yaml

```
@@ -1,6 +1,10 @@
 name: Hugo Build and Deploy
 
-on: push
+on: 
+  pull_request:
+    types: [opened, reopened, closed]
```

```suggestion
    types: [opened, synchronize]
```
i think this is what we use to run on new PRs and new commits on PR?